eq/server/localServer.cpp

```diff
@@ -87,8 +87,11 @@ bool startLocalServer( const std::string& config )
     eq::server::Loader loader;
     eq::server::ServerPtr server;
 
-    if( config.length() > 3 && config.find( ".eqc" ) == config.length() - 4 )
+    if( config.length() > 3 &&
+        config.compare( config.size() - 4, 4, ".eqc" ) == 0 )
```

Comment:
That's what it used to be, but it is very confusing. If the user specified a *.eqc which fails to load this error needs to be reported by letting chooseConfig() fail, otherwise the user will end up with an autoconfig which is not what he wanted.